packages/virtualbox-deb/virtualbox-deb.pacscript

```diff
@@ -5,30 +5,33 @@ pkgver="7.0.16"
 arch=('amd64')
 breaks=("${gives}-bin")
 pkgdesc="VirtualBox is a general-purpose full virtualizer for x86 hardware, targeted at server, desktop and embedded use"
-distro_version= "$(lsb_release -is)~$(lsb_release -cs)"
-url="https://download.virtualbox.org/virtualbox/${pkgver}/virtualbox-7.0_${pkgver}-162802~${distro_version}_amd64.deb"
+codename="${DISTRO#*:}"
```

Comment:
this variable codename can be used (changing as a more clear name) for setting the internal release `162802`